src/main/java/org/breedinginsight/brapi/v2/model/request/query/StudyQuery.java

```diff
@@ -0,0 +1,53 @@
+package org.breedinginsight.brapi.v2.model.request.query;
+
+import io.micronaut.core.annotation.Introspected;
+import lombok.Getter;
+import org.breedinginsight.api.model.v1.request.query.FilterRequest;
+import org.breedinginsight.api.model.v1.request.query.SearchRequest;
+import org.breedinginsight.brapi.v1.model.request.query.BrapiQuery;
+import org.jooq.tools.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Introspected
+public class StudyQuery extends BrapiQuery {
```

Comment:
I think `externalReferenceId` and `externalReferenceSource` should be added to the query param, and probably change the frontend to send the request using those params instead of `trialDbId`